spec/spec_helper.rb

```diff
@@ -26,6 +26,7 @@
   config.include UserHeaderSpecHelper
 
   config.include RequestSpecHelper, :type => :request
+  config.before(:each, :type => :request) { api_version }
```

Comment:
What do you think about if we did something like adding a `spec/support/v1_helper.rb` or something, and then just did:
```ruby
# this goes here in spec/spec_helper.rb
config.include V1Helper, :type => :v1

# spec/support/v1_helper.rb
module V1Helper
  RSpec.configure do |config|
    config.before(:each, :type => :v1) { api("1.0") }
  end
end
```

Then in all the specs instead of declaring the `let(:api_version)`, you just "tag" them with the `:type => [:request, :v1]`?